AzureSearchableSnapshotsIT#testSearchResultsWhenFrozen timed out

**Build scan**:
[scan](https://gradle-enterprise.elastic.co/s/dq5lehlmavc56/tests/:x-pack:plugin:searchable-snapshots:qa:azure:integTestRunner/org.elasticsearch.xpack.searchablesnapshots.AzureSearchableSnapshotsIT/testSearchResultsWhenFrozen#1)

**Repro line**:
```
./gradlew ':x-pack:plugin:searchable-snapshots:qa:azure:integTestRunner' --tests "org.elasticsearch.xpack.searchablesnapshots.AzureSearchableSnapshotsIT.testSearchResultsWhenFrozen" -Dtests.seed=E9C97AC877AA4049 -Dtests.security.manager=true -Dtests.locale=ar-SA -Dtests.timezone=Europe/Kiev -Druntime.java=11
```

**Reproduces locally?**:
Not for me, but I likely don't have the right setup to run it properly.

**Applicable branches**: 7.x and master

**Failure history**:
https://build-stats.elastic.co/goto/3ef7c8c9ff2a663f17d0ec94ad117eb2

**Failure excerpt**:
```
Caused by: java.net.SocketTimeoutException: 60,000 milliseconds timeout on connection http-outgoing-2 [ACTIVE]
```

--- a/x-pack/plugin/searchable-snapshots/qa/azure/build.gradle
+++ b/x-pack/plugin/searchable-snapshots/qa/azure/build.gradle
@@ -63,7 +63,7 @@ integTest {
   dependsOn repositoryPlugin.bundlePlugin
   runner {
     systemProperty 'test.azure.container', azureContainer
-    systemProperty 'test.azure.base_path', azureBasePath + "_searchable_snapshots_tests"
+    systemProperty 'test.azure.base_path', azureBasePath + "_searchable_snapshots_tests_" + BuildParams.testSeed
   }
 }
 

--- a/x-pack/plugin/searchable-snapshots/qa/gcs/build.gradle
+++ b/x-pack/plugin/searchable-snapshots/qa/gcs/build.gradle
@@ -107,7 +107,7 @@ integTest {
   dependsOn repositoryPlugin.bundlePlugin
   runner {
     systemProperty 'test.gcs.bucket', gcsBucket
-    systemProperty 'test.gcs.base_path', gcsBasePath + "_searchable_snapshots_tests"
+    systemProperty 'test.gcs.base_path', gcsBasePath + "_searchable_snapshots_tests" + BuildParams.testSeed
   }
 }
 

--- a/x-pack/plugin/searchable-snapshots/qa/s3/build.gradle
+++ b/x-pack/plugin/searchable-snapshots/qa/s3/build.gradle
@@ -45,7 +45,7 @@ integTest {
   dependsOn repositoryPlugin.bundlePlugin
   runner {
     systemProperty 'test.s3.bucket', s3Bucket
-    systemProperty 'test.s3.base_path', s3BasePath ? s3BasePath + "_searchable_snapshots_tests" : 'base_path'
+    systemProperty 'test.s3.base_path', s3BasePath ? s3BasePath + "_searchable_snapshots_tests" + BuildParams.testSeed : 'base_path'
   }
 }
 

